src/connectionconfig/connectionDialogWebViewController.ts

```diff
@@ -154,19 +155,19 @@ export class ConnectionDialogWebViewController extends ReactWebViewPanelControll
 		}
 	}
 
-	private getActiveFormComponents(): FormComponent[] {
+	private getActiveFormComponents(): FormItemSpec<IConnectionDialogProfile>[] {
```

Comment:
I renamed `FormComponent` to `FormItemSpec` because I thought that there might be confusion about whether this was a React component.